dcr/scenario_utils/check_waagent_log.py

```diff
@@ -93,6 +92,21 @@ def check_waagent_log_for_errors(waagent_log=AGENT_LOG_FILE, ignore=None):
         {
             'message': r"WARNING EnvHandler ExtHandler Dhcp client is not running."
         },
+        # This happens in CENTOS and RHEL when waagent attempt to format and mount the error while cloud init is already doing it
```

Comment:
do we get these errors at all in DCR V2?

Reply:
haven't seen it honestly. I just copied the complete ignore list from v1 and copied it in v2.

Reply:
we wouldn't expect it, right? sounds like a provisioning error. i'd suggest removing it here and we can always add it back if there a reason to do so

Reply:
makes sense, let me remove it